resources/helm/dask-gateway/values.yaml

```diff
@@ -205,10 +205,12 @@ traefik:
       web:
         # The port HTTP(s) requests will be served on
         port: 80
+        nodePort: null
```

Comment:
Odd, wouldn't the default be `nodePort: null` if it's not present? I'm confused why the existing check in the template results in an invalid spec. Will try to give this a closer look tomorrow morning.

Reply:
Yes -- odd. Perhaps the way the templating system works, missing actually evaluates to an empty string, which then doesn't evaluate to "false". Still - presumably if its referred to as if its in "values.yaml", its good to actually have it there. The other option would be to take it out of "values.yaml" and also out of the service template.

Reply:
(keep in mind that this is golang-based, so that python intuitions might not apply. :))